Nos/Views/ReportMenu.swift

```diff
@@ -138,70 +150,62 @@ struct ReportMenuModifier: ViewModifier {
                 return String(localized: .localizable.reportSendToNosConfirmation(category.displayName))
             case .flagPublicly(let category):
                 return String(localized: .localizable.reportFlagPubliclyConfirmation(category.displayName))
```

Comment:
I think we need to change this string to say "flag publicly" instead of "report publicly". I know that isn't necessarily specified in your ticket requirements but you already changed the title of this alert so we should change the description to match.

Reply:
Ok if I look for all "report" references in the localization file and change them to flag accordingly? or should we leave that for another ticket?

Reply:
That's should be fine. Although Josh changed one of them in another PR, so it might create conflicts.